frontend_tests/casper_tests/00-realm-creation.js

```diff
@@ -58,6 +58,7 @@ casper.then(function () {
             password: 'passwordwhichisreallyreallyreallycomplexandnotguessable',
             terms: true,
             realm_in_root_domain: false,
+            time_format: "24",
```

Comment:
I'm curious if we actually need this for the Casper test to pass, or are we just being explicit here?  I assume register can just default to "12" if `time_format` isn't given, correct?